packages/bundle-source/test/no-transforms.test.js

```diff
@@ -11,6 +11,7 @@ test('no-transforms applies no transforms', async t => {
   const entryPath = url.fileURLToPath(
     new URL(`../demo/circular/a.js`, import.meta.url),
   );
+  // @ts-expect-error Property 'endoZipBase64' does not exist on type '{ moduleFormat: "endoScript"; source: string; } | { moduleFormat: "endoZipBase64"; endoZipBase64: string; endoZipBase64Sha512: string; } | { moduleFormat: "nestedEvaluate"; source: string; sourceMap: string; } | { ...; }'.
```

Comment:
urgh, bad return type

Reply:
I think we already do, and specifically that passing `moduleFormat: 'endoZipBase64'` is supposed to obviate this check. The template for `bundleSource` is supposed to infer which overload is called based on that option and narrow the return type.